src/utils/stockDataUtils/getFormattedStockExchange.test.js

```diff
@@ -0,0 +1,30 @@
+import {
+  getFormattedStockExchange
+} from './getFormattedStockExchange';
+import * as selectors from '../../client/selectors';
+
+describe('getFormattedStockExchange', () => {
+  beforeAll(() => {
+    selectors.exchangeSelector = jest.fn();
+  });
+
+  afterAll(() => {
```

Comment:
If this is in afterAll, I don't know if it's even necessary.  Do you know if it's required?